### PR TITLE
Public Release of ICM 0.2.0

### PR DESCRIPTION
**Pull request by jpengar**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:

This is the PR for the public release of ICM 0.2.0

#### Which issue(s) this PR fixes:

Related to https://github.com/camaraproject/IdentityAndConsentManagement/issues/146

#### Special notes for reviewers:

@hdamker CAMARA-AuthN-AuthZ-Concept.md is DEPRECATED and has been removed in this public release as agreed in the past. Please take a look at the CHANGELOG to see if you agree with the way I've tried to track this issue. It is a bit tricky because the document was removed, then restored in RC1, and finally removed now.

#### Changelog input

```
ICM public release 0.2.0
```

#### Additional documentation 

* https://wiki.camaraproject.org/display/CAM/Meta-release+Fall24

### CODE REVIEW

**Review by AxelNennker — DISMISSED** [review]

**Review by sebdewet — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,55 @@
...
 - [v0.1.0 - Initial version](#v010---initial-version)
 
+# r0.2.0
+
+**This is the public release of "Identity And Consent Management" version 0.2.0**
+
+## Please note:
+
+* The content of the release includes the "Identity And Consent Management" approved deliverables in **[documentation](https://github.com/camaraproject/IdentityAndConsentManagement/tree/r0.2.0/documentation)** folder.
+* The document [Authentication and Authorization Concept for Service APIs](https://github.com/camaraproject/IdentityAndConsentManagement/blob/release-0.1.0/documentation/CAMARA-AuthN-AuthZ-Concept.md) was part of the 0.1.0 release. **It is deprecated** and it has been removed after this public release of "Identity and Consent Management" version 0.2.0.
```
Comment (hdamker) [inline CHANGELOG.md:17]: Slight change, as the document is at the time of the merge of this PR no  longer there and it hasn't been removed *after* the release:

```suggestion
* The document [Authentication and Authorization Concept for Service APIs](https://github.com/camaraproject/IdentityAndConsentManagement/blob/release-0.1.0/documentation/CAMARA-AuthN-AuthZ-Concept.md) was part of the 0.1.0 release. **It has been deprecated** and been removed in the course of the public release of "Identity and Consent Management" version 0.2.0.
```

**Comment by hdamker** [comment]
> @hdamker CAMARA-AuthN-AuthZ-Concept.md is DEPRECATED and has been removed in this public release as agreed in the past. Please take a look at the CHANGELOG to see if you agree with the way I've tried to track this issue. It is a bit tricky because the document was removed, then restored in RC1, and finally removed now.

@jpengar Thanks that you remembered about this needed deletion - relevant is here that Commonalities will release the new version 0.4.0 which has no longer the link to the deprecated document. Suggestion above on your disposal.

Beyond that the PR looks good to me. Approval will come after the TSC (hopefully).

**Review by AxelNennker — APPROVED** [review]

**Review by sebdewet — APPROVED** [review]
LGTM

**Comment by AxelNennker** [comment]
Approved by TSC

**Review by hdamker — COMMENTED** [review]
Approved on behalf of Release Management and the TSC (cf. https://wiki.camaraproject.org/x/pYBFAg).

**Pull request merged**